interceptors/Mementifier.cfc

```diff
@@ -238,11 +238,16 @@ component{
 
             // Verify Nullness
             thisValue = isNull( thisValue ) ? (
-                structKeyExists( thisMemento.defaults, item ) ? thisMemento.defaults[ item ] : $mementifierSettings.nullDefaultValue
-            ) : thisValue;
-
+				arrayContainsNoCase( thisMemento.defaults.keyArray(), item ) ?
+					( isNull( thisMemento.defaults[ item ] ) ? javacast( "null", "" ) : thisMemento.defaults[ item ] ) :
+					$mementifierSettings.nullDefaultValue
+			) : thisValue;
+			
+			if ( isNull( thisValue ) ) {
+				result[ item ] = javacast( "null", "" );
```

Comment:
I think you can just `continue;` here and skip the `if` block.  I don't think a re-scoping is necessary, since you're doing an explicit null cast above.

Reply:
NM. I guess you' haven't scoped the `result` key, yet.